lib/everypolitician/pull_request/report/memberships.rb

```diff
@@ -13,11 +13,16 @@ def removed
         private
 
         def before_h
-          @before_h ||= before.memberships.map { |m| [m.document, m] }.to_h
+          @before_h ||= before.memberships.map { |m| [document_without_sources(m.document), m] }.to_h
         end
 
         def after_h
-          @after_h ||= after.memberships.map { |m| [m.document, m] }.to_h
+          @after_h ||= after.memberships.map { |m| [document_without_sources(m.document), m] }.to_h
+        end
+
+        def document_without_sources(doc)
+          doc.delete(:sources)
+          doc
```

Comment:
Yep, good points! I've added `Hash#except` in https://github.com/everypolitician/everypolitician-pull_request/pull/22/commits/aefe8420a608a5f4b5758f92b5a5fc3198b86f85 and switched to using it in https://github.com/everypolitician/everypolitician-pull_request/pull/22/commits/87164fc283069e7eb7a194363d4d387e3b2e9bcf.